engine-nextgen/src/main/kotlin/graphql/nadel/validation/NadelInterfaceValidation.kt

```diff
@@ -0,0 +1,32 @@
+package graphql.nadel.validation
+
+import graphql.nadel.validation.NadelSchemaValidationError.MissingConcreteTypes
+import graphql.schema.GraphQLInterfaceType
+
+internal class NadelInterfaceValidation {
+    fun validate(
+        schemaElement: NadelServiceSchemaElement,
+    ): List<NadelSchemaValidationError> {
+        return if (schemaElement.overall is GraphQLInterfaceType) {
+            validateHasImplementations(schemaElement)
+        } else {
+            emptyList()
+        }
+    }
+
+    private fun validateHasImplementations(
+        schemaElement: NadelServiceSchemaElement,
+    ): List<NadelSchemaValidationError> {
+        val underlyingSchema = schemaElement.service.underlyingSchema
+        val underlyingInterfaceType = schemaElement.underlying as GraphQLInterfaceType
+        val underlyingImplementations = underlyingSchema.getImplementations(underlyingInterfaceType)
```

Comment:
Not 100% sure this logic is correct. Could someone define an interface implementation in the overall schema only? They could… but who would? and for now nobody does… so let's just do this until someone complains.